.circleci/config.yml

```diff
@@ -689,7 +689,7 @@ jobs:
 
   osx_package:
     macos:
-      xcode: "10.0.0"
+      xcode: "10.0.0" # 10.0.0 is macOS 10.13.6
```

Comment:
@dincho I added this comment while trying to reproduce on CI the rocksdb compilation failure I was getting.